phpstan.neon.dist

```diff
@@ -33,6 +33,11 @@ parameters:
 			count: 1
 			path: admin/admin-base.php
 
+		-
```

Comment:
Could you please add a comment about this ignored error even if it's ununderstable ? 🤪

Reply:
Done in [8b381b2](https://github.com/polylang/polylang/pull/1041/commits/8b381b2611127599a88e49dc09dec6e8d2040dab) and it's perfect.